File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] arr = new int[3];

        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
        }

        Arrays.sort(arr);

        if (arr[0] + arr[1] + arr[2] == 180) {
            if (arr[0] == arr[2]) {
                System.out.println("Equilateral");
            } else if (arr[0] == arr[1] && arr[1] != arr[2] || arr[1] == arr[2] && arr[0] != arr[1]) {
                System.out.println("Isosceles");
            } else {
                System.out.println("Scalene");
            }
        } else {
            System.out.println("Error");
        }
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
import sys

# sys.stdin = open("input.txt", "r")

nums = [0] * 3

nums[0] = int(sys.stdin.readline().strip())
nums[1] = int(sys.stdin.readline().strip())
nums[2] = int(sys.stdin.readline().strip())

nums = sorted(nums)

if sum(nums) == 180:
    if nums[0] == nums[2]:
        print('Equilateral')
    elif (nums[0] == nums[1] and nums[1] != nums[2]) or (nums[0] != nums[1] and nums[1] == nums[2]):
        print('Isosceles')
    else:
        print('Scalene')
else:
    print('Error')